servicemesh-install/topics/install-overview.adoc

```diff
@@ -5,40 +5,14 @@ The {ProductName} installation process creates two different projects (namespace
 
 * istio-operator project (1 pod)
 * istio-system project (17 pods)
+* kiali-operator project (1 pod)
+* observability project (1 pod)
 
 You first create a Kubernetes _operator_. This operator defines and monitors a _custom resource_ that manages the deployment, updating, and deletion of the {ProductShortName} components.
 
 Depending on how you define the custom resource file, you can install one or more of the following components when you install the {ProductShortName}:
 
-* *Istio* - based on the  open source https://istio.io/[Istio] project, lets you connect, secure, control, and observe the microservices that make up your applications.
-* *Jaeger* - based on the opensource https://www.jaegertracing.io/[Jaeger] project, lets you perform tracing to monitor and troubleshoot transactions in complex distributed systems.
-* *Kiali* - based on the opensource https://www.kiali.io/[Kiali] project, Kiali provides observability for your service mesh. Using Kiali lets you view configurations, monitor traffic, and view and analyze traces in a single console.
-* *Launcher* - based on the opensource  http://fabric8.io/[fabric8] community, this integrated development platform helps you build cloud native applications and microservices. {ProductName} includes several boosters that let you explore features of the {ProductShortName}.
+* *Istio* - based on the open source https://istio.io/[Istio] project, lets you connect, secure, control, and observe the microservices that make up your applications.
+* *Jaeger* - based on the open source https://www.jaegertracing.io/[Jaeger] project, lets you perform tracing to monitor and troubleshoot transactions in complex distributed systems.
+* *Kiali* - based on the open source https://www.kiali.io/[Kiali] project, provides observability for your service mesh. Using Kiali lets you view configurations, monitor traffic, and view and analyze traces in a single console.
 
-During the installation the operator creates an Ansible job that runs an Ansible playbook that performs the following installation and configuration tasks automatically:
```

Comment:
I know all this has changed, but it would still be nice to include a list of what exactly is being installed.  It's helpful for troubleshooting purposes (to verify that everything has been installed that we're expecting to be installed).  @brian-avery or @tvieira  could you help with that information?